controllers/mover/syncthing/mover.go

```diff
@@ -490,6 +496,9 @@ func (m *Mover) ensureDeployment(ctx context.Context, dataPVC *corev1.Persistent
 			},
 		}
 
+		// Cluster-wide proxy settings
```

Comment:
Isn't this redundant w/ L475?

Reply:
Good catch! That was my mistake  :grimacing: